.rubocop.yml

```diff
@@ -17,18 +21,43 @@ Bundler/DuplicatedGem:
 Bundler/OrderedGems:
   Enabled: true
 
+Layout/FirstArrayElementIndentation:
+  EnforcedStyle: consistent
+
+Layout/LineLength:
+  Max: 100
+
+Lint/Debugger:
+  Exclude:
+    - "config/application.rb"
+
+Lint/ScriptPermission:
+  Exclude:
+    - "Rakefile"
+
+Metrics/AbcSize:
+  Max: 35
+  Exclude:
+    - "spec/**/*"
+
 Metrics/BlockLength:
+  Max: 50
   Exclude:
-    - "spec/**/*_spec.rb"
-    - "config/environments/**/*.rb"
+    - "config/**/*"
+    - "spec/**/*"
```

Comment:
Did you run into anything specific here that requires ignoring more than just the *_spec.rb files?

Reply:
same for the configs... shouldn't they also contain readable code?